search-service/src/test/kotlin/com/egm/stellio/search/csr/service/ContextSourceUtilsTests.kt

```diff
@@ -0,0 +1,149 @@
+package com.egm.stellio.search.csr.service
+
+import com.egm.stellio.search.common.config.SearchProperties
+import com.egm.stellio.search.csr.model.Mode
+import com.egm.stellio.search.temporal.service.TemporalPaginationService
+import com.egm.stellio.shared.model.CompactedEntity
+import com.egm.stellio.shared.util.*
+import com.egm.stellio.shared.util.JsonLdUtils.JSONLD_TYPE_TERM
+import com.egm.stellio.shared.util.JsonLdUtils.JSONLD_VALUE_TERM
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_DATASET_ID_TERM
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_OBSERVED_AT_TERM
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.mockk.every
+import io.mockk.mockkObject
+import io.mockk.verify
+import kotlinx.coroutines.test.runTest
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.springframework.boot.context.properties.EnableConfigurationProperties
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+
+@ActiveProfiles("test")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE, classes = [TemporalPaginationService::class])
```

Comment:
`TemporalPaginationService`?

Reply:
Actually, you don't even need a Spring Boot test here, a simple JUnit test should be enough?